.github/workflows/compile.yml

```diff
@@ -13,14 +13,6 @@ jobs:
         dotnet-version: |
           5.0.x
           6.0.x
-    - uses: actions/cache@v2
-      with:
-        path: ~/.nuget/packages
```

Comment:
You can cache/restore packages (without lock files) according to the version.
Without cache `dotnet` will download all dependencies in every CI job.